src/ServiceContainer/SymfonyExtension.php

```diff
@@ -90,13 +91,12 @@ public function configure(ArrayNodeDefinition $builder): void
         $builder
             ->addDefaultsIfNotSet()
                 ->children()
+                    ->scalarNode('env_file')->defaultValue('.env')->end()
                     ->arrayNode('kernel')
-                        ->addDefaultsIfNotSet()
+                    ->addDefaultsIfNotSet()
                         ->children()
-                            ->scalarNode('bootstrap')->defaultValue('app/autoload.php')->end()
                             ->scalarNode('path')->defaultValue('app/AppKernel.php')->end()
                             ->scalarNode('class')->defaultValue('AppKernel')->end()
-                            ->scalarNode('env')->defaultValue('test')->end()
```

Comment:
I would keep the `env` config instead of using .env

Reply:
I am not sure it is a good idea, you can configure two different kernel environments. For instance, `APP_ENV=test` in the `.env` and `env: in_memory` in the behat profile.

Reply:
Same here, BC break.

Reply:
I'd like to support both the current format and the one with `.env`, same like Symfony 3.3/3.4.